src/main/java/com/ordwen/odailyquests/quests/player/progression/checkers/AbstractEntityChecker.java

```diff
@@ -41,9 +41,8 @@ public void setPlayerQuestProgression(Player player, EntityType entityType, Stri
 
                 if (abstractQuest instanceof EntityQuest quest) {
 
-                    if (quest.getEntityTypes() == null) isRequiredEntity = true;
+                    if (quest.getEntityTypes() == null || quest.getEntityTypes().isEmpty()) isRequiredEntity = true;
```

Comment:
The list is never created if the quest section does not contain the `required_entity` part... But it's smart to add this if someone leaves the part empty :)